lib/pageflow/global_config_api.rb

```diff
@@ -46,12 +46,12 @@ def configure(&block)
     def config_for(target)
       config = build_config do |c|
         c.enable_features(target.enabled_feature_names)
+        call_entry_type_config_blocks(c, target) if target.respond_to?(:type_name)
```

Comment:
We still need to handle the case, when we want to know all file types in the system ([example](https://github.com/codevise/pageflow/blob/master/lib/pageflow/ability_mixin.rb#L179)). To make it safe to enable configure blocks of multiple entry types, I guess, we will need to make [`FileTypes`](https://github.com/codevise/pageflow/blob/0fa72a514f620f169737a4eb908630fd8d0064da/lib/pageflow/file_types.rb#L10) resilient against duplicate registrations.

Reply:
I'd argue that because `FileTypes`' `each` uses `uniq(&:model)` (see [1]), `FileTypes` doesn't need any more resilience against duplicate registrations

[1] https://github.com/codevise/pageflow/blob/0fa72a514f620f169737a4eb908630fd8d0064da/lib/pageflow/file_types.rb#L20